### ISSUE TITLE
Option to Ignore Paper

### ISSUE THREAD

**Issue by MaxvanHaastrecht** [asreview/asreview#328]
## Feature Request

**Is your feature request related to a problem? Please describe.**
Duplicate papers can be contained in the set of papers used for ASReview. This can be the result of using multiple sources, where not each source provides the exact same data (meaning it is more difficult to detect duplicates). Currently, the only options in ASReview are to include or exclude a paper. For a duplicate paper, you want to do neither.

**Describe the solution you'd like**
A solution to this problem would be an option to mark a paper as duplicate. If there are other use cases for this option, you may want to call this option something generic such as 'ignore'. I would suggest to then completely retrain the model without the ignored paper and also exclude it from any future calculations. It should still be shown in the eventual output, since it was a decision made by the researcher that they would like to document.

**Describe alternatives you've considered**
In terms of duplicate detection, a feature within ASReview could be considered, to help out researchers (like myself) who struggle to properly remove duplicates. I cannot necessarily think of any other alternatives.

**Teachability, Documentation, Adoption, Migration Strategy**
When a reviewer realizes that the paper he is currently looking at is a duplicate, he can choose the 'ignore' button. A pop-up explaining the consequences of their decision is displayed and the reviewer is asked to agree with the consequences. The model is retrained and the paper is ignored in further calculations.

**Comment by J535D165** [comment]
Sounds like a very useful feature and not that hard to implement after #322 is implemented. I like to have the 2 options: `skip (this is a duplicate)` and `skip (I can't decide)`. Or something like this. 

If anyone is interested in implementing this, please drop a message over here.

**Comment by Rensvandeschoot** [comment]
@J535D165 I agree with the first option (duplicate), but the second option is more problematic... what will you technically do? Show the next record in the cue? Or, train the model again without this record? I would strongly recommend the user of our software to make a decision. Maybe it helps if we add a comment field (see also [this feature](https://github.com/asreview/asreview/issues/377) request)

**Comment by J535D165** [comment]
Good one. We need some more discussion on this. But I do like the idea to start with starting with the option to add notes. Thereafter, we can think about the skip/ignore option. Let's keep this issue open for now, maybe someone is interested in working on this. That's fine.

**Comment by Rensvandeschoot** [comment]
The option to add notes has been implemented in version 1, and [@rohitgarud](https://github.com/rohitgarud/asreview-notes-export) has built an [extension](https://github.com/rohitgarud/asreview-notes-export) to extract the notes from the project file.

**Comment by Rensvandeschoot** [comment]
[@j0sien](https://github.com/j0sien) has raised the issue of skipping records in https://github.com/asreview/asreview/discussions/1297. So, let's open the discussion again :-)
